### ISSUE TITLE
KIE: Segmentation fault (core dumped) 

### ISSUE THREAD

**Issue by botastark** [PaddlePaddle/PaddleOCR#8256]
I can't infer (tools/infer_kie_token_ser.py) from finetuned ser_vi_layoutxlm_xfund_zh_udml.
I use ser_vi_layoutxlm_xfund_zh.yml for inference similar as  config used for training ser_vi_layoutxlm_xfund_zh_udml.yml
after loading the model, it says Segmentation fault (core dumped)

[2022/11/09 16:02:33] ppocr INFO: Architecture : 
[2022/11/09 16:02:33] ppocr INFO:     Backbone : 
[2022/11/09 16:02:33] ppocr INFO:         checkpoints : output/ser_vi_layoutxlm_xfund_zh_udml_play/best_accuracy
[2022/11/09 16:02:33] ppocr INFO:         mode : vi
[2022/11/09 16:02:33] ppocr INFO:         name : LayoutXLMForSer
[2022/11/09 16:02:33] ppocr INFO:         num_classes : 89
[2022/11/09 16:02:33] ppocr INFO:         pretrained : True
[2022/11/09 16:02:33] ppocr INFO:     Transform : None
[2022/11/09 16:02:33] ppocr INFO:     algorithm : LayoutXLM
[2022/11/09 16:02:33] ppocr INFO:     model_type : kie
[2022/11/09 16:02:33] ppocr INFO: Eval : 
[2022/11/09 16:02:33] ppocr INFO:     dataset : 
[2022/11/09 16:02:33] ppocr INFO:         data_dir : train_data_play/val
[2022/11/09 16:02:33] ppocr INFO:         label_file_list : ['train_data_play/val/val.txt']
[2022/11/09 16:02:33] ppocr INFO:         name : SimpleDataSet
[2022/11/09 16:02:33] ppocr INFO:         transforms : 
[2022/11/09 16:02:33] ppocr INFO:             DecodeImage : 
[2022/11/09 16:02:33] ppocr INFO:                 channel_first : False
[2022/11/09 16:02:33] ppocr INFO:                 img_mode : RGB
[2022/11/09 16:02:33] ppocr INFO:             VQATokenLabelEncode : 
[2022/11/09 16:02:33] ppocr INFO:                 algorithm : LayoutXLM
[2022/11/09 16:02:33] ppocr INFO:                 class_path : train_data_play/class_list.txt
[2022/11/09 16:02:33] ppocr INFO:                 contains_re : False
[2022/11/09 16:02:33] ppocr INFO:                 order_method : tb-yx
[2022/11/09 16:02:33] ppocr INFO:                 use_textline_bbox_info : True
[2022/11/09 16:02:33] ppocr INFO:             VQATokenPad : 
[2022/11/09 16:02:33] ppocr INFO:                 max_seq_len : 512
[2022/11/09 16:02:33] ppocr INFO:                 return_attention_mask : True
[2022/11/09 16:02:33] ppocr INFO:             VQASerTokenChunk : 
[2022/11/09 16:02:33] ppocr INFO:                 max_seq_len : 512
[2022/11/09 16:02:33] ppocr INFO:             Resize : 
[2022/11/09 16:02:33] ppocr INFO:                 size : [224, 224]
[2022/11/09 16:02:33] ppocr INFO:             NormalizeImage : 
[2022/11/09 16:02:33] ppocr INFO:                 mean : [123.675, 116.28, 103.53]
[2022/11/09 16:02:33] ppocr INFO:                 order : hwc
[2022/11/09 16:02:33] ppocr INFO:                 scale : 1
[2022/11/09 16:02:33] ppocr INFO:                 std : [58.395, 57.12, 57.375]
[2022/11/09 16:02:33] ppocr INFO:             ToCHWImage : None
[2022/11/09 16:02:33] ppocr INFO:             KeepKeys : 
[2022/11/09 16:02:33] ppocr INFO:                 keep_keys : ['input_ids', 'bbox', 'attention_mask', 'token_type_ids', 'image', 'labels']
[2022/11/09 16:02:33] ppocr INFO:     loader : 
[2022/11/09 16:02:33] ppocr INFO:         batch_size_per_card : 2
[2022/11/09 16:02:33] ppocr INFO:         drop_last : False
[2022/11/09 16:02:33] ppocr INFO:         num_workers : 4
[2022/11/09 16:02:33] ppocr INFO:         shuffle : False
[2022/11/09 16:02:33] ppocr INFO: Global : 
[2022/11/09 16:02:33] ppocr INFO:     cal_metric_during_train : False
[2022/11/09 16:02:33] ppocr INFO:     class_path : train_data_play/class_list.txt
[2022/11/09 16:02:33] ppocr INFO:     distributed : False
[2022/11/09 16:02:33] ppocr INFO:     epoch_num : 50
[2022/11/09 16:02:33] ppocr INFO:     eval_batch_step : [0, 19]
[2022/11/09 16:02:33] ppocr INFO:     infer_img : train_data_play/val/20223000527387379915_Librettodicircolazione_page_0-fs8.png
[2022/11/09 16:02:33] ppocr INFO:     kie_det_model_dir : None
[2022/11/09 16:02:33] ppocr INFO:     kie_rec_model_dir : None
[2022/11/09 16:02:33] ppocr INFO:     log_smooth_window : 10
[2022/11/09 16:02:33] ppocr INFO:     print_batch_step : 10
[2022/11/09 16:02:33] ppocr INFO:     save_epoch_step : 2000
[2022/11/09 16:02:33] ppocr INFO:     save_inference_dir : None
[2022/11/09 16:02:33] ppocr INFO:     save_model_dir : ./output/ser_vi_layoutxlm_xfund_zh_udml_play
[2022/11/09 16:02:33] ppocr INFO:     save_res_path : output/ser_res_play/
[2022/11/09 16:02:33] ppocr INFO:     seed : 2022
[2022/11/09 16:02:33] ppocr INFO:     use_gpu : True
[2022/11/09 16:02:33] ppocr INFO:     use_visualdl : True
[2022/11/09 16:02:33] ppocr INFO: Loss : 
[2022/11/09 16:02:33] ppocr INFO:     key : backbone_out
[2022/11/09 16:02:33] ppocr INFO:     name : VQASerTokenLayoutLMLoss
[2022/11/09 16:02:33] ppocr INFO:     num_classes : 89
[2022/11/09 16:02:33] ppocr INFO: Metric : 
[2022/11/09 16:02:33] ppocr INFO:     main_indicator : hmean
[2022/11/09 16:02:33] ppocr INFO:     name : VQASerTokenMetric
[2022/11/09 16:02:33] ppocr INFO: Optimizer : 
[2022/11/09 16:02:33] ppocr INFO:     beta1 : 0.9
[2022/11/09 16:02:33] ppocr INFO:     beta2 : 0.999
[2022/11/09 16:02:33] ppocr INFO:     lr : 
[2022/11/09 16:02:33] ppocr INFO:         epochs : 50
[2022/11/09 16:02:33] ppocr INFO:         learning_rate : 5e-05
[2022/11/09 16:02:33] ppocr INFO:         name : Linear
[2022/11/09 16:02:33] ppocr INFO:         warmup_epoch : 2
[2022/11/09 16:02:33] ppocr INFO:     name : AdamW
[2022/11/09 16:02:33] ppocr INFO:     regularizer : 
[2022/11/09 16:02:33] ppocr INFO:         factor : 0.0
[2022/11/09 16:02:33] ppocr INFO:         name : L2
[2022/11/09 16:02:33] ppocr INFO: PostProcess : 
[2022/11/09 16:02:33] ppocr INFO:     class_path : train_data_play/class_list.txt
[2022/11/09 16:02:33] ppocr INFO:     name : VQASerTokenLayoutLMPostProcess
[2022/11/09 16:02:33] ppocr INFO: Train : 
[2022/11/09 16:02:33] ppocr INFO:     dataset : 
[2022/11/09 16:02:33] ppocr INFO:         data_dir : train_data_play/train
[2022/11/09 16:02:33] ppocr INFO:         label_file_list : ['train_data_play/train/train.txt']
[2022/11/09 16:02:33] ppocr INFO:         name : SimpleDataSet
[2022/11/09 16:02:33] ppocr INFO:         ratio_list : [1.0]
[2022/11/09 16:02:33] ppocr INFO:         transforms : 
[2022/11/09 16:02:33] ppocr INFO:             DecodeImage : 
[2022/11/09 16:02:33] ppocr INFO:                 channel_first : False
[2022/11/09 16:02:33] ppocr INFO:                 img_mode : RGB
[2022/11/09 16:02:33] ppocr INFO:             VQATokenLabelEncode : 
[2022/11/09 16:02:33] ppocr INFO:                 algorithm : LayoutXLM
[2022/11/09 16:02:33] ppocr INFO:                 class_path : train_data_play/class_list.txt
[2022/11/09 16:02:33] ppocr INFO:                 contains_re : False
[2022/11/09 16:02:33] ppocr INFO:                 order_method : tb-yx
[2022/11/09 16:02:33] ppocr INFO:                 use_textline_bbox_info : True
[2022/11/09 16:02:33] ppocr INFO:             VQATokenPad : 
[2022/11/09 16:02:33] ppocr INFO:                 max_seq_len : 512
[2022/11/09 16:02:33] ppocr INFO:                 return_attention_mask : True
[2022/11/09 16:02:33] ppocr INFO:             VQASerTokenChunk : 
[2022/11/09 16:02:33] ppocr INFO:                 max_seq_len : 512
[2022/11/09 16:02:33] ppocr INFO:             Resize : 
[2022/11/09 16:02:33] ppocr INFO:                 size : [224, 224]
[2022/11/09 16:02:33] ppocr INFO:             NormalizeImage : 
[2022/11/09 16:02:33] ppocr INFO:                 mean : [123.675, 116.28, 103.53]
[2022/11/09 16:02:33] ppocr INFO:                 order : hwc
[2022/11/09 16:02:33] ppocr INFO:                 scale : 1
[2022/11/09 16:02:33] ppocr INFO:                 std : [58.395, 57.12, 57.375]
[2022/11/09 16:02:33] ppocr INFO:             ToCHWImage : None
[2022/11/09 16:02:33] ppocr INFO:             KeepKeys : 
[2022/11/09 16:02:33] ppocr INFO:                 keep_keys : ['input_ids', 'bbox', 'attention_mask', 'token_type_ids', 'image', 'labels']
[2022/11/09 16:02:33] ppocr INFO:     loader : 
[2022/11/09 16:02:33] ppocr INFO:         batch_size_per_card : 2
[2022/11/09 16:02:33] ppocr INFO:         drop_last : False
[2022/11/09 16:02:33] ppocr INFO:         num_workers : 4
[2022/11/09 16:02:33] ppocr INFO:         shuffle : True
[2022/11/09 16:02:33] ppocr INFO: profiler_options : None
[2022/11/09 16:02:33] ppocr INFO: train with paddle 2.3.2 and device Place(gpu:0)
W1109 16:02:33.859344  5909 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 7.5, Driver API Version: 11.4, Runtime API Version: 10.2
W1109 16:02:33.863689  5909 gpu_resources.cc:91] device: 0, cuDNN Version: 8.5.
[2022/11/09 16:02:36] ppocr INFO: resume from output/ser_vi_layoutxlm_xfund_zh_udml_play/best_accuracy
[2022-11-09 16:02:37,441] [    INFO] - Already cached /root/.paddlenlp/models/layoutxlm-base-uncased/sentencepiece.bpe.model
[2022-11-09 16:02:38,004] [    INFO] - tokenizer config file saved in /root/.paddlenlp/models/layoutxlm-base-uncased/tokenizer_config.json
[2022-11-09 16:02:38,022] [    INFO] - Special tokens file saved in /root/.paddlenlp/models/layoutxlm-base-uncased/special_tokens_map.json
Segmentation fault (core dumped)

**Comment by tangjielong928** [comment]
hi, I also meet this problem, how can u solve it?
